generated/google_cloud_rpc/lib/service_client.dart

```diff
@@ -0,0 +1,249 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:convert';
+
+import 'package:google_cloud_protobuf/protobuf.dart';
+import 'package:google_cloud_rpc/rpc.dart';
+import 'package:http/http.dart' as http;
+
+import 'src/versions.dart';
+
+export 'dart:typed_data' show Uint8List;
+
+export 'src/web.dart'
+    if (dart.library.io) 'src/vm.dart'
+    show httpClientFromApiKey;
+
+const String _clientKey = 'x-goog-api-client';
+
+// ignore: prefer_const_declarations
+final String _clientName = 'gl-dart/$clientDartVersion gax/$gaxVersion';
```

Comment:
Just double checking that we still want our client name to contain 'gax' (`gl-dart/0.x.0 gax/0.2.0`).